wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1194,11 +1198,118 @@ class PanelNode(BlockInclusionNode):
 register.tag("panel", PanelNode.handle)
 
 
-class FieldNode(BlockInclusionNode):
+class RawFormattedFieldNode(BlockInclusionNode):
+    content_var = "rendered_field"
     template = "wagtailadmin/shared/field.html"
 
 
-register.tag("field", FieldNode.handle)
+register.tag("rawformattedfield", RawFormattedFieldNode.handle)
```

Comment:
This is a bit hard to read as a single word all in lowercase. Could we add underscores?

```suggestion
register.tag("raw_formatted_field", RawFormattedFieldNode.handle)
```

Reply:
I thought that might be a little bit contentious... I went with the underscore-less version because it's consistent with how BlockInclusionNode generates the end tag by prepending `end` with no delimiter. As it stands, that would give us `endraw_formatted_field` rather than the expected `end_raw_formatted_field`. If we're changing that, we should probably update BlockInclusionNode to allow overriding that.

(Django also seems to have a slight preference for the no-underscore version in its own tags - e.g. `blocktranslate` - although it isn't really consistent.)